openhcl/fixed_pool_alloc/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
looks like you've got some dupe'd copyright headers (likely fallout from #172)

Reply:
Nice, did not see this, and also I think CI gate is failing for copyright headers, will fix.